src/services/dashboard.js

```diff
@@ -7,11 +7,15 @@ import { getAllMonthsWithinRange } from '../utils/moment_constants';
 const { endpoints } = window;
 
 function scrollUntilEmpty(data) {
-  const endpoint = `${endpoints.elasticsearch}/_search/scroll?scroll=1m`;
+  const url = `/_search/scroll?scroll=1m`;
```

Comment:
I don't think `url` is used, and it's redundant with `indices` below.

Reply:
@npalaska ?